values.yaml

```diff
@@ -243,4 +274,124 @@ supernode:
+      # default: 5
+      eliminationLimit: 5
+
+      # FailureCountLimit is the failure count limit set in supernode for dfget client.
+      # When a dfget client takes part in the peer network constructed by supernode,
+      # supernode will command the peer to start distribution task.
+      # When dfget client fails to finish distribution task, the failure count of client
+      # increases by 1. When failure count of client reaches to FailureCountLimit(default 5),
+      # dfget client will be moved to blacklist of supernode to stop playing as a peer.
+      # default: 5
+      failureCountLimit: 5
+
+      # LinkLimit is set for supernode to limit every piece download network speed.
+      # default: 20 MB, in format of G(B)/g/M(B)/m/K(B)/k/B, pure number will also be parsed as Byte.
+      linkLimit: 20M
+
+      # SystemReservedBandwidth is the network bandwidth reserved for system software.
+      # default: 20 MB, in format of G(B)/g/M(B)/m/K(B)/k/B, pure number will also be parsed as Byte.
+      systemReservedBandwidth: 20M
+
+      # MaxBandwidth is the network bandwidth that supernode can use.
+      # default: 200 MB, in format of G(B)/g/M(B)/m/K(B)/k/B, pure number will also be parsed as Byte.
+      maxBandwidth: 200M
+
+      # Whether to enable profiler
+      # default: false
+      enableProfiler: false
+
+      # Whether to open DEBUG level
+      # default: false
+      debug: false
+
+      # FailAccessInterval is the interval time after failed to access the URL.
+      # If a task failed to be downloaded from the source, it will not be retried in the time since the last failure.
+      # default: 3m
+      failAccessInterval: 3m
+
+      # gc related
+
+      # GCInitialDelay is the delay time from the start to the first GC execution.
+      # default: 6s
+      gcInitialDelay: 6s
+
+      # GCMetaInterval is the interval time to execute GC meta.
+      # default: 2m0s
+      gcMetaInterval: 2m
+
+      # TaskExpireTime when a task is not accessed within the taskExpireTime,
+      # and it will be treated to be expired.
+      # default: 3m0s
+      taskExpireTime: 3m
+
+      # PeerGCDelay is the delay time to execute the GC after the peer has reported the offline.
+      # default: 3m0s
+      peerGCDelay: 3m
+
+      # GCDiskInterval is the interval time to execute GC disk.
+      # default: 15s
+      gcDiskInterval: 15s
+
+      # YoungGCThreshold if the available disk space is more than YoungGCThreshold
+      # and there is no need to GC disk.
+      # default: 100GB
+      youngGCThreshold: 100G
+
+      #  FullGCThreshold if the available disk space is less than FullGCThreshold
+      #  and the supernode should gc all task files which are not being used.
+      #  default: 5GB
+      fullGCThreshold: 5G
+
+      # IntervalThreshold is the threshold of the interval at which the task file is accessed.
+      # default: 2h0m0s
+      IntervalThreshold: 2h
```

Comment:
Do we need such detailed config template file here? 
I preferring removing the comments and putting these comments in a separate config doc.

Reply:
Agree.

Reply:
Done.